lib/access_import/access_import.rb

```diff
@@ -7,28 +9,53 @@ def initialize(path)
     @sprache_pro_hauptperson.add_other_accessors(@sprachen, @sprach_kenntnisse)
     @einsatz_orte.add_other_accessors(@plz)
     @haupt_person.add_other_accessors(@plz, @laender, @sprache_pro_hauptperson)
+    @import_user = User.find_by(email: 'aoz_access_importer@example.com') || make_import_superadmin
   end
 
   def make_clients
     transformer = ClientTransform.new(@begleitete, @haupt_person, @familien_rollen)
     make(@personen_rolle.all_clients, transformer, Client) do |client, personen_rolle|
       client = personen_rollen_create_update_conversion(client, personen_rolle)
-      client.user_id = User.where(role: 'superadmin').first.id
-      if personen_rolle[:d_Rollenende]
-        client.state = Client::FINISHED
-        " -- Client was set to state finished at #{personen_rolle[:d_Rollenende]}  ----"
-      end
+      client.user_id = @import_user.id
+      client.state = handle_client_state(personen_rolle)
     end
   end
 
+  def handle_client_state(personen_rolle)
+    return Client::FINISHED if personen_rolle[:d_Rollenende]
+    return Client::ACTIVE if personen_rolle[:d_Rollenende].nil?
+    Client::REGISTERED
```

Comment:
I would prefer a multiline `if/elsif` or even `case` but that's just me :)

Reply:
I would usually not agree, but I'll be a good boy this time.... 😉